src/Suin/RSSWriter/Item.php

```diff
@@ -90,6 +93,47 @@ public function appendTo(ChannelInterface $channel)
         return $this;
     }
 
+    public function customElement($tagName, $content = null, $options = array())
+    {
+        $this->customElements[] = array(
+            'tagName' => $tagName,
+            'content' => $content,
+            'options' => $options,
+        );
+
+        return $this;
+    }
+
+    /**
+     * @param SimpleXMLElement $xml
+     * @param array $item
+     */
+    protected function transformCustomElement(SimpleXMLElement $xml, $item)
+    {
+        $options = $item['options'];
+
+        $namespace = null;
+        if (array_key_exists('namespace', $options)) {
+            $namespace = $options['namespace'];
+        }
+
+        if (array_key_exists('cdata', $options) && $options['cdata'] === true) {
+            // SimpleXMLElement does not support CDATA transformation
+            $element = $xml->addChild($item['tagName'], null, $namespace);
+            $element = dom_import_simplexml($element);
+            $elementOwner = $element->ownerDocument;
+            $element->appendChild($elementOwner->createCDATASection($item['content']));
+        } else {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The method transformCustomElement uses an else expression. Else is never necessary and you can simplify the code to work without else.](https://www.codacy.com/app/suinyeze/php-rss-writer/file/2259992134/issues/source?bid=3151561&fileBranchId=3183140#l126)